### ISSUE TITLE
Connection issue with FAD + Lyrics Plus (webnowplaying?)

### ISSUE THREAD

**Issue by Chroma67** [khanhas/spicetify-cli#1401]
## ℹ Computer information

- Spotify version: 1.1.76.447.g11f432d8
- Spicetify version: 2.8.4

## 📝 Provide detailed reproduction steps (if any)

1. Enable Lyrics Plus integration in Full App Display Config
2. Refresh FAD (Tried refreshing spotify as well)

### ✔️ Expected result

Lyrics on the right of the Album Cover

### ❌ Actual result

No Lyrics loaded all though Lyrics Plus still fetches lyrics fine.

## 📷 Screenshots

![image](https://user-images.githubusercontent.com/69115700/148862154-ed9f92e8-2115-4b30-a849-741cdb3dee1d.png)
![image](https://user-images.githubusercontent.com/69115700/148862191-db5bee50-e307-4357-81de-699e5123af24.png)

**Comment by Chroma67** [comment]
![image](https://user-images.githubusercontent.com/69115700/148862276-de559c90-844f-4a57-b2b3-b93223041962.png)

**Comment by afonsojramos** [comment]
@CharlieS1103 Who added the Lyrics to the FAD? Maybe that person can help.

**Comment by 4310V343k** [comment]
I can have a similar issue, but lyrics plus is not working for me at all, including the FAD
https://i.imgur.com/TWxz6hU.png

**Comment by Chroma67** [comment]
@4310V343k have you checked console with developer tools?

**Comment by 4310V343k** [comment]
> @4310V343k have you checked console with developer tools?

Yes, and i can see it's a different error. Will open a different issue

**Comment by Chroma67** [comment]
anything on this issue?

**Comment by Chroma67** [comment]
@afonsojramos sorry for the second ping but it seems like this issues has not been resolved

**Comment by afonsojramos** [comment]
I'll try to take a look on Sunday, but can't promise anything 🧐

**Comment by Chroma67** [comment]
thank you

**Comment by Chroma67** [comment]
I changed my theme because I had made slight alterations myself to test if it was the problem. It didn't fix anything
